### ISSUE TITLE
Shard mapping rooms

### ISSUE THREAD

**Issue by cpryor6** [rpherbig/dr-scripts#3138]
It looks like there is a duplicate entry for room 2611, when you 'go home' manually from 2611 you end up in a new room that also says it's 2611

[go2]>go corr
You run southeast.
[Shard, Emerald Lane]
Patches of bright green grass grows around the pebbled street.  The street narrows as it empties out to the southeast, the hypnotic, spiral pattern of its blue and green stones swirling up to the feet of the Great Tower.  The sound of hawkers and the call of a street faire entices passersby to the north.
Obvious paths: southeast, northwest.
Room Number: 2504
>
[go2]>go gard door
You run southeast.
[Shard, North Entrance of The Great Tower]
A high plaster wall marks the end of Diamond Street, the road narrowing down until it splits into three parts.  One leads up to a sweeping marble archway, while the other two paths circumvent the barrier.  A shining silver tower rises up from beyond the walls, the brisk snap of banners hanging with all the colors of the Elothean Houses bright against the gleaming liquid silver surface.
Obvious paths: north, northeast, east, west, northwest.
Room Number: 2503
>
You run north.
[go2]>go path
[Shard, Diamond Street]
A splash of green from flourishing gardens and the glint of light off crystal windows offers a tantalizing view of the homes of the privileged of Shard.  Houses alternately built with the Elven mind for arches and spindly, steeple-like points take up residence with the Elothean touch of round buildings with faceted domes.  Groups of citizens trod down the lapis-hued street, entering or exiting to the north via an ivory and brass gate that divides the inner city from its outer, more common sister.  You also see a posh Elven home with a chased silver sign on it.
Obvious paths: south.
Room Number: 2611
>
What were you referring to?
>
What were you referring to?
>
What were you referring to?
>
[go2: restarting script...]
[go2: ETA: 0:00:00 (3 rooms to move through)]
[go2]>go corr
What were you referring to?
>
[go2: move: failed]
[go2: restarting script...]
[go2: ETA: 0:00:00 (3 rooms to move through)]
[go2]>go corr
What were you referring to?
>
[go2: move: failed]
>
[go2: restarting script...]

**Comment by Tarhish** [comment]
Yeah I see the problem.  The pathing was messed up around the healer there.

I reattached the map pieces and snipped those wrong-ways.  Should be working the next time you download the map.
